modules/oxide/src/less/theme/components/toolbar-button/toolbar-button.less

```diff
@@ -73,9 +73,41 @@
     text-transform: @toolbar-button-text-transform;
     width: @toolbar-button-width;
 
+    @media (forced-colors: active) {
+      &:hover {
+        filter: brightness(70%);
+      }
+
+      &.tox-tbtn--active,
```

Comment:
I don't know if this is enough of a difference for `active` 🤔

Reply:
it's not, this one is really just a place holder, was planning to add outline to this one as well